client/components/ui/sidebar/Sidebar.vue

```diff
@@ -0,0 +1,161 @@
+<script lang="ts" setup>
+import { useStorage } from '@vueuse/core'
+
+const { t } = useI18n()
+const { isDark, toggle } = useDark()
+
+const themeIcon = computed(() => isDark.value ? 'i-tabler-sun' : 'i-tabler-moon')
+const themeName = computed(() => isDark.value ? t('sidebar.light_mode') : t('sidebar.dark_mode'))
+
+const sidebar = useStorage('sidebar', { open: true })
+const sidebarOpen = computed(() => sidebar.value.open)
```

Comment:
actually an insanely good example where reactive is good

you can do this

```suggestion
const sidebar = toReactive(useStorage('sidebar', { open: true }))
```

, delete the sidebarOpen computed ref, and manipulate sidebar.open as you want

so for example, the toggle now is

```js
function toggleSidebar() { return sidebar.open = !sidebar.open }
```

thanks antfu

https://vueuse.org/shared/toReactive/